include/swift/AST/Module.h

```diff
@@ -210,6 +210,7 @@ class PackageUnit: public DeclContext {
   /// Equality check via package name instead of pointer comparison.
   /// Returns false if the name is empty.
   bool isSamePackageAs(PackageUnit *other) {
+    if (!other) return false;
```

Comment:
nitpick: could you move `return false` to the next line for better debuggability?